components/button/share/src/index.js

```diff
@@ -0,0 +1,83 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+import {Socialtwitter, Socialfacebook, Commentsquare, Envelopeclosed} from '@schibstedspain/sui-svgiconset'
+
+const getOnClickHandle = src => () => window.open(src)
+
+const parseHttpSchemeOfUrl = (urlToEval) => {
+  const isUrlValid = urlToEval.match(/http:\/\/|https:\/\//)
+  return !isUrlValid ? `http://${urlToEval}` : urlToEval
```

Comment:
I think string functions should be much quicker here.

```
return urlToEval.startsWith('http://') || urlToEval.startsWith('https://')
 ? urlToEval
 : `http://${urlToEval}`
```

I think the polyfill is in the default list but if not indexOf can make the same thing

Reply:
Done!